src/Containers/FeaturedProfilesCard.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import styled from 'styled-components';
+import { AddUser } from '@styled-icons/entypo/AddUser';
+import { flex } from '../Utils/Mixins';
+
+export interface IProfile {
+    image: string;
+    initials: string;
+    id: number;
+}
+
+export interface IFeaturedProfilesCardProps {
+    profileData: IProfile[];
+    alt: string;
+    width?: number;
+    height?: number;
+}
+
+export const FeaturedProfilesCard: React.FC<IFeaturedProfilesCardProps> = ({
+    alt,
+    profileData,
+    width = 100,
+    height = 100,
+}): React.ReactElement => {
+    const getProfileCircles = (): any => {
+        const PROFILE_PICTURE_LIMIT = 3;
+        const PROFILE_INITIALS_START_INDEX = 2;
+        const PROFILE_INITIALS_END_INDEX = 5;
+        const REMAINING_PROFILES_INDEX = 5;
+        const REMAINING_PROFILES_VALUE = profileData.length - 5;
```

Comment:
This can stay inside though, just lowercase since it's not a TRUE constant, it's a dynamic variable so lowercase all